Doxygen/src/EventRecorder.md

```diff
@@ -252,11 +251,48 @@ The memory required for the Event Recorder data is calculated with the formula:
 ```
 In this example we configure `0x800` bytes (as it is easier) which can hold more than 64 records.
 
-### Create memory region{#evr_create_mem}
+### Create memory region {#evr_create_mem}
 
-To setup this uninitialized RAM, use either a
-<a href="https://developer.arm.com/documentation/101754/latest/armlink-Reference/Scatter-loading-Features" target="_blank">linker script</a>
-or configure it in µVision following these steps:
+To setup this uninitialized RAM, use either \ref evr_create_mem_ls or \ref evr_create_mem_uv procedure.
+
+#### Create memory region using linker script {#evr_create_mem_ls}
+
+If the linker script does not contain provisions for uninitialized memory section then, for respective toolchain, add the necessary section like described below:
+
+##### Arm Compiler 6 {#evr_create_mem_ls_ac6}
```

Comment:
We could drop '6' since it applies also to legacy 5. Looks better if we just use "Arm Compiler"